codecoverage.py

```diff
@@ -202,6 +204,7 @@ def main():
     parser.add_argument("src_dir", action="store", nargs=nargs, default=default_src_dir, help="Path to the source directory")
     parser.add_argument("branch", action="store", nargs='?', default=default_branch, help="Branch on which jobs ran")
     parser.add_argument("commit", action="store", nargs='?', default=default_commit, help="Commit hash for push")
+    parser.add_argument("output_path", action="store", nargs='?', help="Path to store report")
```

Comment:
No need for this, for a user of the command line it doesn't matter where the info file is stored. So in the command line case, you can always use `output.info`.